stanford_news.features.field_instance.inc

```diff
@@ -31,6 +31,13 @@ function stanford_news_field_default_field_instances() {
         'type' => 'text_default',
         'weight' => 4,
       ),
+      'stanford_news_extras' => array(
```

Comment:
This is a setting in the manage fields display. Because we have the extras fields available, it captures some of the field settings. This happened we added the cap view mode to person.